Add `charts` to templates (#11054)
netdata/netdata: health/REFERENCE.md

| [`hosts`](#alarm-line-hosts)                        | no              | Which hostnames will run this alarm.                                                  |
| [`plugin`](#alarm-line-plugin)                      | no              | Restrict an alarm or template to only a certain plugin.                                             |
| [`module`](#alarm-line-module)                      | no              | Restrict an alarm or template to only a certain module.                                             |

> | [`charts`](#alarm-line-charts)                      | no              | Restrict an alarm or template to specified charts.                                             |

Reviewer: ```suggestion
| [`charts`](#alarm-line-charts)                      | no              | Restrict an alarm or template to only certain charts.                                             |
```

to allign with other descriptions, sounds good?

Reviewer: Also, should we make it clear that it works with templates only? 

> Restrict a template to only certain charts.

Author: I decided to call attention for this, because it does not make sense to use this feature with `alarm`.

module: isc_dhcpd
```

#### Alarm line `charts`

The `charts` line filters which chart this alarm should apply to. It is only available with [`template`](#alarm-line-alarm-or-template).
The value is a space-separated list of [simple patterns](/libnetdata/simple_pattern/README.md). For
example, you can create a template that applies on `disk.svctm` (Average Service Time) context, but you exclude

> alarms for the disk `sdb`:

Reviewer: @joelhans please check the wording

> The `charts` line filters which chart this alarm should apply to. It is only available with [`template`](#alarm-line-alarm-or-template).
> The value is a space-separated list of [simple patterns](/libnetdata/simple_pattern/README.md). For
> example, you can create a template that applies on `disk.svctm` (Average Service Time) context, but you exclude
> alarms for the disk `sdb`:

Reviewer 2: ```suggestion
The `charts` line filters which chart this alarm should apply to. It is only available on entities using the [`template`](#alarm-line-alarm-or-template) line.
The value is a space-separated list of [simple patterns](/libnetdata/simple_pattern/README.md). For
example, a template that applies to the `disk.svctm` (Average Service Time) context, but excludes the disk `sdb` from alarms:
```

Author: Thank you @joelhans !

```yaml
    on: disk.svctm

> charts: !*sdb*

Reviewer: ```suggestion
charts: !*sdb* *
```

Reviewer: > If it is not matched by either positive or negative patterns, it is denied at the end.

> ```

Reviewer 2: Could this example include the `template:` line for full context?
